gitlabform/processors/project/tags_processor.py

```diff
@@ -33,13 +36,20 @@ def _process_configuration(self, project_and_group: str, configuration: dict):
                             elif "user_id" in config:
                                 user_ids.add(config["user_id"])
                             elif "user" in config:
-                                user_ids.add(self.gitlab._get_user_id(config["user"]))
+                                users_by_username = self.gl.users.list(
+                                    username=config["user"]
+                                )
+                                if len(users_by_username) == 0:
+                                    raise NotFoundException(
+                                        "No users found when searching for username '%s'"
+                                        % config["user"]
+                                    )
+                                user_ids.add(users_by_username[0]["id"])
                             elif "group_id" in config:
                                 group_ids.add(config["group_id"])
                             elif "group" in config:
-                                group_ids.add(
-                                    self.gitlab._get_group_id(config["group"])
-                                )
+                                group_by_name = self.gl.groups.get(config["group"])
```

Comment:
Similar to users, lets capture exception about group if it doesn't exist.

Reply:
Changed to `gitlab_group`, and caught raised `GitlabGetError`

Reply:
Not sure if I missed it buf didn't see the exception being caught. Are we just expectung the code to reach the exception from the python-gitlab libray and not need to catch it here? Maybe we don't need to catch it here?

Reply:
Yeh, I'm letting the `GitlabGetError` exception from either `self.gl.groups.get(config["group"])` or `self._get_user_by_username(config["user"])` bubble up to eventually be caught by the top-level `except` in `_process_configuration()`. If neither the user or group can be found the rest of the config can't be safely applied so just bails out at that point.